### PR TITLE
Document parsing Delwaq results.

### PR DESCRIPTION
**Pull request by evetion**
And plot some, including:
- Utility to add tracers to existing models
- Adds default Initial (basin state) and UserDemand tracers
- Make some fancy plots

fraction_example

fraction_spatial_example

### CODE REVIEW

**Comment by evetion** [comment]
Hmm, I forgot that we can't plot results, as we haven't run Delwaq in our docs.

**Comment by evetion** [comment]
> Hmm, I forgot that we can't plot results, as we haven't run Delwaq in our docs.

After a discussion with @visr on this, we decided to upload the results of the Delwaq test to minio, and use that in our plots. It will lag behind somewhat, but that should be not matter too much for documentation purposes.

For that purposes I've added a upload_benchmark.py script (named to mirror the get_benchmark one) and improved both scripts to be more configurable (using personal keys) and argparse, and adjusted the TC configuration to match. The scripts should all use environment variables IMHO, but I'll leave that to another issue, as an implementation of DVC will supersede those improvements.

File: utils/get_benchmark.py [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
-myClient = Minio(minioServer, access_key=accessKey, secret_key=secretKey)
 
-objects = myClient.list_objects(bucketName, prefix=pathToFolder, recursive=True)
+parser = argparse.ArgumentParser(
+    description="Download a folder (recursively) from the MinIO server"
+)
+parser.add_argument("folder", help="The path to download in the MinIO server")
+parser.add_argument(
+    "--accesskey",
+    help="The secret key to access the MinIO server",
```
Comment (visr) [inline utils/get_benchmark.py:16]:
```suggestion
    help="The access key to access the MinIO server",
```

File: utils/upload_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+minioServer = "s3.deltares.nl"
+bucketName = "ribasim"
+
+parser = argparse.ArgumentParser(description="Upload a file to the MinIO server")
+parser.add_argument("source", type=Path, help="The source file to upload")
+parser.add_argument("destination", help="The destination file in the MinIO server")
+parser.add_argument(
+    "--accesskey",
+    help="The secret key to access the MinIO server",
```
Comment (visr) [inline utils/upload_benchmark.py:16]:
```suggestion
    help="The access key to access the MinIO server",
```

**Review by visr — APPROVED** [review]
Looks good to me!

**Pull request merged**